components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik, Field, Form } from 'formik';
+import { Flex, Box } from '@rebass/grid';
+import { assign } from 'lodash';
+import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+
+import { H1, P } from '../../Text';
+import Container from '../../Container';
+import Illustration from '../../home/HomeIllustration';
+import StyledCheckbox from '../../StyledCheckbox';
+import StyledInput from '../../StyledInput';
+import StyledInputField from '../../StyledInputField';
+import StyledInputGroup from '../../StyledInputGroup';
+import StyledButton from '../../StyledButton';
+import MessageBox from '../../MessageBox';
+
+import { Router } from '../../../server/pages';
+
+class CreateCollectiveForm extends React.Component {
+  static propTypes = {
+    error: PropTypes.string,
+    host: PropTypes.object,
+    query: PropTypes.object,
+    collective: PropTypes.object,
+    loading: PropTypes.bool,
+    onSubmit: PropTypes.func,
+    intl: PropTypes.object.isRequired,
+    onChange: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+
+    const collective = { ...(props.collective || {}) };
```

Comment:
You should be able to spread an undefined object.
```js
const collective = { ...props.collective }; // {}
```

Reply:
TIL! Thanks 🙌